iothub/device/src/DirectMethod/DirectMethodRequest.cs

```diff
@@ -69,37 +84,28 @@ public DirectMethodRequest()
         /// </summary>
         protected internal byte[] Payload { get; set; }
 
+        /// <summary>
+        /// The convention to use with the direct method payload.
+        /// </summary>
+        [JsonIgnore]
+        protected internal PayloadConvention PayloadConvention { get; set; }
+
         /// <summary>
         /// Method timeout, in seconds.
         /// </summary>
         [JsonProperty("responseTimeoutInSeconds", NullValueHandling = NullValueHandling.Ignore)]
-        protected internal int? ResponseTimeoutInSeconds => ResponseTimeout.HasValue && ResponseTimeout > TimeSpan.Zero
+        internal int? ResponseTimeoutInSeconds => ResponseTimeout.HasValue && ResponseTimeout > TimeSpan.Zero
```

Comment:
Users will set this via the public properties these read from. This is for serialization only.